src/main/java/com/cleanroommc/modularui/value/sync/PanelSyncManager.java

```diff
@@ -79,8 +82,8 @@ public void onOpen() {
     public void onClose() {
         this.closeListener.forEach(listener -> listener.accept(getPlayer()));
         for (String name : this.subPanels.keySet()) {
-            SyncHandler sh = this.modularSyncManager.getMainPSM().syncHandlers.remove(name);
-            this.modularSyncManager.getMainPSM().reverseSyncHandlers.remove(sh);
+            SyncHandler sh = this.subPanels.get(name).getSyncManager().syncHandlers.remove(name);
+            this.subPanels.get(name).getSyncManager().reverseSyncHandlers.remove(sh);
```

Comment:
What is the reason for this change???

Reply:
panel sync handlers are stored in the parent sync manager of the subpanel, which isn't guaranteed to be the main PSM

Reply:
Why did you change that? I made the psm only be registered to the main psm, because that never gets closed.
Also when you open the 2nd panel on the test tile then close and then open it again an error is thrown